app/policies/service_plan_policy.rb

```diff
@@ -0,0 +1,11 @@
+class ServicePlanPolicy < ApplicationPolicy
```

Comment:
@eclarizio Could this inherit from PortfolioItemPolicy

Reply:
@mkanoor It could, but I don't think we should. The `#create` method is different and `PortfolioItemPolicy` doesn't have `#update_modified?` or `#reset?` methods so we would be re-writing those anyway.